packages/data-models/src/lexeme.js

```diff
@@ -50,6 +50,40 @@ class Lexeme {
     this.addInflections(inflections)
     this.meaning = meaning || new DefinitionSet(this.lemma.word, this.lemma.languageID)
     this.disambiguated = false
+    this.selectedInflection = null
+  }
+
+  /**
+   * Set the selected inflection for a lexeme which has had its
+   * inflections disambiguated
+   * @param {Inflection} inflection the selected inflection
+   */
+  setSelectedInflection (inflection) {
+    this.selectedInflection = inflection
+  }
+
+  /**
+   * Get the selected inflection for a lexeme which has had its
+   * inflections disambiguated
+   * @returns {Inflection} (or null if none is selected)
+   */
+  getSelectedInflection () {
+    return this.selectedInflection
+  }
+
+  /**
+   * Gets the selected inflection formatted for display
+   * (returns an array because the display is grouped by feature
+   * but there should only be one inflection in the array)
+   * @returns {Array} if no selected inflection the array will be empty
+   */
+  getSelectedInflectionForDisplay () {
```

Comment:
I'm wondering if it's better to remove `forDisplay` from the name because it refers to the UI and we want data models to be UI-unaware. Maybe to rename it to something that would reflect how inflections be grouped?

Reply:
ok, yes you're probably right here. I hate the whole grouping code and would like to rip it out and do it better, which is probably why I avoided the name here. But it is what it is.